config/environment.js

```diff
@@ -18,6 +18,11 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    contentSecurityPolicy: {
```

Comment:
Hello @ericdouglas,
the above looks like a duplicate key declaration to me, with the part on [line 7](https://github.com/ericdouglas/ember-simple-blog/pull/9/files#diff-695693c5dabaffb064bde5b060195ff7R7). :)